dags/dbt_examples/copy_store_failures_bigquery.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+from airflow.utils.dates import datetime
```

Comment:
```suggestion
from dateime import datetime
```
Let's use the `datetime` package itself.